model/src/main/java/com/epam/esm/model/parameters/CertificateParameter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.model.parameters;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CertificateParameter {
+    TAG_NAME("tagName", TableColumnName.TAG_DAO_NAME),
+    CERTIFICATE_NAME("name", TableColumnName.CERTIFICATE_DAO_NAME),
+    CERTIFICATE_DESCRIPTION("description", TableColumnName.CERTIFICATE_DAO_DESCRIPTION),
+    CREATE_DATE("createDate", TableColumnName.CERTIFICATE_DAO_CREATE_DATE);
+
+    String name;
+    String tableColumn;
+
+    CertificateParameter(String name, String tableColumn) {
+        this.name = name;
+        this.tableColumn = tableColumn;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getTableColumn() {
+        return tableColumn;
+    }
+
+    public static Optional<CertificateParameter> getByName (String name) {
```

Comment:
what is the purpose of returning Optional here?

Reply:
also there should no be a space after method name (follow Java Code Convention)

Reply:
done